src/controllers/PartyController.js

```diff
@@ -5,9 +5,11 @@ class PartyController {
 
   static getParty(req, res) {
     const party = parties.find(c => c.id === parseInt(req.params.id, 10));
```

Comment:
Using "c" to name a variable here is wrong, use a more meaningful and descriptive  name, e.p "party"